src/releng_docs/index.rst

```diff
@@ -34,7 +34,7 @@ Install Nix:
 
 .. code-block:: bash
 
-    % curl https://raw.githubusercontent.com/mozilla-releng/services/master/nix/setup.sh | sh
+    % curl https://raw.githubusercontent.com/mozilla-releng/services/master/nix/setup.sh | bash
```

Comment:
We could use `$SHELL` if we can verify the script works under the most common ones. Pipes are evaluated in reverse, so this will provide a nice early error message that the shell we need doesn't exist.